src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -198,6 +199,8 @@
     protected YesOrNo isRespondent1;
     private List<IdValue<Bundle>> caseBundles;
     private RespondToClaim respondToClaim;
+    private RespondToClaim respondToAdmittedClaim;
```

Comment:
Looking at the use of this variable, I think your tests could use respondToClaim variable , rather than creating another variable respondToAdmittedClaim.

Reply:
About the respondToAdmittedClaim, I have created a new variable in the CaseData respondToAdmittedClaim in order to handle the front end value passing. Therefore, I add this for testing on the "Admit Part of the Claim" route

Reply:
The use of this might be okay, please think over it.